.github/workflows/zerodowntime_tests.yml

```diff
@@ -0,0 +1,453 @@
+
+          echo "##[set-output name=HELM_CHART_NAME;]$(echo ${HELM_CHART_NAME})"
+          echo "##[set-output name=HELM_SERVICE_HELM_CHART_NAME;]$(echo ${HELM_SERVICE_HELM_CHART_NAME})"
+          echo "##[set-output name=JMETER_SERVICE_HELM_CHART_NAME;]$(echo ${JMETER_SERVICE_HELM_CHART_NAME})"
+
+      # setup cloud provider kubernetes instance
+
+      - name: Install and start GKE cluster
+        if: env.CLOUD_PROVIDER == 'GKE'
+        env:
+          GKE_VERSION: ${{ matrix.PLATFORM_VERSION }}
+          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_RESTRICTED_SERVICE_KEY }}
+          GCLOUD_PROJECT_NAME: ${{ secrets.GCLOUD_PROJECT_NAME }}
+          CLOUDSDK_COMPUTE_ZONE: "us-east1-b"
+          CLOUDSDK_REGION: "us-east1"
+          GKE_KUBECONFIG: ${{ secrets.INTEGRATION_TEST_KUBECONFIG }}
+          BRANCH: ${{ steps.determine_branch.outputs.BRANCH}}
+        id: install_gke
+        run: |
+          export CLUSTER_NAME_NIGHTLY=${{ matrix.CLUSTER_NAME }}
+          echo "Setting up GCloud CLI"
+          export OS_TYPE="linux"
+          
+          echo ${GCLOUD_SERVICE_KEY} | base64 --decode > ~/gcloud-service-key.json
+          export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+          export CLOUDSDK_CORE_DISABLE_PROMPTS=1;
+          
+          gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+          gcloud --quiet config set project "$GCLOUD_PROJECT_NAME"
+          gcloud --quiet config set container/cluster "$CLUSTER_NAME_NIGHTLY"
+          gcloud --quiet config set compute/zone "${CLOUDSDK_COMPUTE_ZONE}"
+          
+          echo "GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS}" >> $GITHUB_ENV
+          
+          echo "Setting up kubectl"
+          echo "Setting KUBECONFIG to $PWD/.kubeconfig"
+          
+          echo "$GKE_KUBECONFIG" >> "$PWD/.kubeconfig"
+          chmod 600 "$PWD/.kubeconfig"
+          export KUBECONFIG="$PWD/.kubeconfig"
+          
+          echo "KUBECONFIG=${KUBECONFIG}" >> $GITHUB_ENV
+          
+          echo "Setting kube context..."
+          kubectl config use-context "$CLUSTER_NAME_NIGHTLY"
+          
+          echo "##[set-output name=CLUSTER_NAME_NIGHTLY;]$(echo ${CLUSTER_NAME_NIGHTLY})"
+
+      - name: Install Istio
+        if: env.CLOUD_PROVIDER != 'minishift-on-GHA' && env.CLOUD_PROVIDER != 'GKE' # no need to install istio on minishift or GKE (pre-installed)
+        run: test/utils/install_istio.sh
+
+      - name: Install Keptn
+        id: keptn_install
+        timeout-minutes: 10
+        env:
+          HELM_CHART_NAME: ${{ steps.extract_helm_chart_name.outputs.HELM_CHART_NAME }}
+          USE_RESOURCE_SERVICE: ${{ matrix.USE_RESOURCE_SERVICE }}
+        run: |
+          echo "Installing Keptn on cluster"
+          
+          echo "{}" > creds.json # empty credentials file
+          
+          
+          echo "::group::Keptn Installation Log"
+          if [[ $RUN_AIRGAPPED_TEST == "true" ]]; then
```

Comment:
the airgapped code can be removed here

Reply:
so no need to change between keptn and keptndev in the other cases?